Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM python:3.12.0b4-slim
```

Comment:
We're currently on 3.11 in this file, but I think we should stay there?

Reply:
Actually, should this be using 3.11-bookworm? We're mostly using it but not always.

Reply:
Consistency is good; where are we using bookworm?

Reply:
Ah, saw below. Yeah, maybe try it and see what happens?

Reply:
Yeah, I don't know if there's a reason it's bookworm in one place but not the other, but since IIRC we are actually running this on the platform bookworm is meant for, I expect it should be a useful switch for us? I'll doublecheck the convo about where upstream started using bookworm for if there was a technical reason for the discrepancy versus just "one dockerfile is more hidden than the other".

Reply:
I checked upstream and they're using bookworm for dev-tools but not the main dockerfile. Let's stay aligned with them, and do regular 3.11 here?

Reply:
I was not able to find any discussion of why, because "bookworm" is a very common typo of "bookwyrm" and I'm not sure which python version Bookwyrm was on when Bookworm was introduced.

Reply:
Aha! I found it after all. https://github.com/bookwyrm-social/bookwyrm/issues/3190 was fixed in https://github.com/bookwyrm-social/bookwyrm/pull/3216 by using Bookworm for dev-tools. Let's stay consistent with upstream, since that shouldn't be necessary here.